sdk/daml-lf/interpreter/src/main/scala/com/daml/lf/speedy/SBuiltinFun.scala

```diff
@@ -1152,16 +1137,14 @@ private[lf] object SBuiltinFun {
     *    -> a
     */
 
-  final case class SBFetchAny(optTargetTemplateId: Option[TypeConName]) extends UpdateBuiltin(2) {
+  final case class SBFetchTemplate(templateId: TypeConName) extends UpdateBuiltin(2) {
```

Comment:
Should we maybe call it `targetTemplateId` or `dstTemplateId` (as it is now called in `fetchTemplate`) to emphasize that some upgrade may take place?

Reply:
In my point of view, here it is pretty clear, it is the template Id of the Fetch. 
Note that in function `fetchTemplate`, where it can be helpful to distinguish between the two template IDs,  I keep dstTmplId`.